ssv/spectest/tests/runner/consensus/post_finish.go

```diff
@@ -48,7 +48,7 @@ func PostFinish() *tests.MultiMsgProcessingSpecTest {
 							Root:       sha256.Sum256(testingutils.TestSyncCommitteeContributionConsensusDataByts),
 						}), nil),
 				},
-				PostDutyRunnerStateRoot: "d800b0a2047ef1caa04a7a97d803558f5f5e3c6efb5c6f0e28c94023d3676609",
+				PostDutyRunnerStateRoot: "dbc5a425e55618d83b245a71e1858419f8cd6d4b1b1022b68403da001c950601",
```

Comment:
What should I change here? 

As I understand it creates a finished (decided) runner and then processes a commit message expecting an invalid proposal error because it doesn't process any proposal messages. 

Should I add a proposal message before the commit message or delete this test or change the logic in another way?

Reply:
If that's what it's testing then it's not the purpose of the test.
The purpose is to test a consensus message post finish

Reply:
@alonmuroch Could you please have a look at this test again? I pushed changes in https://github.com/bloxapp/ssv-spec/pull/162/commits/c6ec2c5809a06d4643c25a00e3fdf30ea8f0f259 and https://github.com/bloxapp/ssv-spec/pull/162/commits/5a5604d0ed695681d4fe79a503d4c59a39dcd05a. Does it look correct now?

Reply:
the test tests a commit message after runner finished, we don't send a commit msg but rather a proposal so it doesn't test what the test should be testing

Reply:
@alonmuroch Should the test be testing a **multi-signer** commit message after the runner has finished? With a regular commit message we'd get a `did not receive proposal for this round` error

Reply:
I pushed the changes that use multi-signer commit message, could you please review?